jobs/adspert_python_dev.html

```diff
@@ -0,0 +1,40 @@
+---
+title: Python Software Developer
+company: Adspert (Bidmanagement GmbH)
+url: https://www.adspert.net/en/jobs/python-software-developer-mf/
+location: Berlin, Germany
+contract: permanent
+contact:
+    name: Karl Bartel
+    email: jobs@adspert.net
+created: !!timestamp '2017-05-15'
+tags:
+  - python
+  - sql
+---
+We are looking for a **Python Software Developer (m/f)**
```

Comment:
That is probably a problem caused by translation. In Germany, most companies add (m/f) to avoid legal action, since the term "software developer" is male by default. I guess simply removing (m/f) in English would be a good solution, right?